velox/common/base/Status.h

```diff
@@ -192,21 +194,33 @@ class [[nodiscard]] Status {
   //   auto st1 = Status::UserError("my error"):
   //   auto st2 = Status::TypeError("my other error"):
 
-  /// Return an error status for out-of-memory conditions.
+  /// Return an error status for user errors.
+  static Status UserError() {
```

Comment:
to prevent you from specifying this for each error code, couldn't you just forward all arguments to `fromArgs()`, and provide a `fromArgs()` specialization that takes only the error code?

Reply:
@pedroerp Updated. Would you take another look?